### DISCUSSION TITLE
Date in Page

### DISCUSSION THREAD

**Discussion by emdefikri** [mecha-cms/mecha#116] [Q&A] [answered]
Hai Mas Taufik, saya memasang ekstensi language, saya ganti lokasi GMT Jakarta dan bahasa id kok data tanggal posting hilang ya? Saya lihat renderisasi akhir di HTML, tanggal masih tampil di dalam atribut HTML, tapi nilai yang diapit tag time tidak tampil.

[https://maulidadzulfikri.me/jurnal/terjemahan-sastra-sebagai-tafsir](https://maulidadzulfikri.me/jurnal/terjemahan-sastra-sebagai-tafsir)

**Comment by taufik-nurrohman** [comment, marked as answer]
Di tema bawaan, tampilan format tanggal dibuat supaya menyesuaikan bahasa yang aktif:

https://github.com/mecha-cms/mecha/blob/c825d9fe5be913f5be2068046ff89c9a20da914c/lot/layout/page.php#L10

Kalau mau dibuat spesifik ke format `en` tinggal diganti saja jadi:

~~~ php
<?= $page->time->en; ?>
~~~

Format `id` harusnya memang ditambahkan juga di ekstensi _language_, cuma Saya masih belum mantap di mana sebaiknya meletakkan dukungan format ini:

~~~ php
Time::_('id', '%A, %d %B %Y');
~~~

**Referensi:**
 - [`Time::_()`](https://mecha-cms.com/reference/class/time/_)
 - [`$time->i()`](https://mecha-cms.com/reference/class/time/i)
